analyses/bfa/notebooks/bfa_seas_bavg_precip_corr_obsfor.md

```diff
@@ -61,6 +61,7 @@ sys.path.append(path_mod)
 from src.indicators.drought.config import Config
 
 from src.indicators.drought.iri_rainfallforecast import get_iri_data
+from src.utils_general.statistics import _get_return_period_function_analytical, _get_return_period_function_empirical
```

Comment:
Commenting here but referring to above lines:
- Line 2: Through jupytext you can choose to not save the metadata, this helps other people avoid kernel confusion when they start up the notebook
- Line 49: Did you add myst_nb to the requirements? I'm getting some version conflicts with the current version of nbconvert required

Reply:
- Hmm I had to save the metadata cause I was experimenting with rendering it with jupyter book. Will at least remove the kernelspec now. 
- Good point, I will add it. Though this is really experimentation, not sure how clean we want to keep the requirements file.

Reply:
> not sure how clean we want to keep the requirements file

Yeah, that is something that has been on my mind. Maybe we should make a separate requirements file for the analysis directory? Something to add to my running list of issues to add to issues...